examples/06-custom-schema/08-non-editable-block/index.html

```
@@ -0,0 +1,14 @@
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)